src/api/uploads/MultiputUpload.js

```diff
@@ -494,11 +494,7 @@ class MultiputUpload extends BaseMultiput {
             }: { buffer: ArrayBuffer, readCompleteTimestamp: number } = await this.readFile(reader, blob);
             const sha256ArrayBuffer = await digest('SHA-256', buffer);
             const sha256 = btoa(
-                [].reduce.call(
-                    new Uint8Array(sha256ArrayBuffer),
-                    (data, byte) => data + String.fromCharCode(byte),
-                    ''
-                )
+                [].reduce.call(new Uint8Array(sha256ArrayBuffer), (data, byte) => data + String.fromCharCode(byte), '')
```

Comment:
Did you mean to include this in this PR?

Reply:
prettier strikes again